sdk/cardboard.cc

```diff
@@ -321,6 +321,13 @@ void CardboardHeadTracker_getPose(CardboardHeadTracker* head_tracker,
   std::memcpy(orientation, &out_orientation[0], 4 * sizeof(float));
 }
 
+void CardboardHeadTracker_recenter(CardboardHeadTracker* head_tracker){
+if (CARDBOARD_IS_NOT_INITIALIZED() || CARDBOARD_IS_ARG_NULL(head_tracker)) {
```

Comment:
Please fix indentation. I recommend using ClangFormat to automate the code formatting.